src/pages/FeedbackDetail/FeedbackDetail.tsx

```
@@ -40,101 +40,41 @@ export interface FeedbackDetailResponse {
 }
 
 const FeedbackDetailPage = () => {
+  const navigate = useNavigate();
   const { id } = useParams();
-  const { data } = useFetch<FeedbackDetailResponse>(`/api/answers/${id}`);
-
-  const question = data?.question;
-  const feedback = data?.feedback;
+  if (!id) return <Navigate to={ROUTE_PATH.ARCHIVE} replace />;
 
+  const { question, feedback, memo, starred = false, level = 0, answerText } = useAnswers(id);
```

Biome (2.1.2)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

Hooks should not be called after an early return.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)